airgun/views/oscapcontent.py

```diff
@@ -0,0 +1,90 @@
+from widgetastic.widget import (
+    FileInput,
+    Text,
+    TextInput,
+    View,
+)
+from widgetastic_patternfly import BreadCrumb
+
+from airgun.views.common import (
+    BaseLoggedInView,
+    SatTab,
+    SearchableViewMixin,
+)
+from airgun.widgets import (
+    ActionsDropdown,
+    MultiSelect,
+    SatTable,
+)
+
+
+class SCAPContentsView(BaseLoggedInView, SearchableViewMixin):
+    title = Text("//h1[text()='SCAP Contents']")
+    new = Text("//a[contains(@href, 'scap_contents/new')]")
+    table = SatTable(
+        './/table',
+        column_widgets={
+            'Title': Text('./a'),
+            'Actions': ActionsDropdown("./div[contains(@class, 'btn-group')]"),
+        }
+    )
+
+    @property
+    def is_displayed(self):
+        return self.browser.wait_for_element(
+            self.title, exception=False) is not None
+
+
+class SCAPContentCreateView(BaseLoggedInView):
+    submit = Text('//input[@name="commit"]')
+    cancel = Text('//a[text()="Cancel"]')
+
+    @View.nested
+    class file_upload(SatTab):
+        TAB_NAME = 'File Upload'
+        title = TextInput(id='scap_content_title')
+        scap_file = FileInput(id='scap_content_scap_file')
+
+    @View.nested
+    class locations(SatTab):
+        resources = MultiSelect(id='ms-scap_content_location_ids')
+
+    @View.nested
+    class organizations(SatTab):
+        resources = MultiSelect(id='ms-scap_content_organization_ids')
+
+    @property
+    def is_displayed(self):
+        return self.browser.wait_for_element(
+            self.file_upload.title, exception=False) is not None
+
+
+class SCAPContentEditView(BaseLoggedInView):
+    breadcrumb = BreadCrumb()
+    submit = Text('//input[@name="commit"]')
+    cancel = Text('//a[text()="Cancel"]')
+
+    @property
+    def is_displayed(self):
+        breadcrumb_loaded = self.browser.wait_for_element(
+            self.breadcrumb, exception=False)
+        return (
+                breadcrumb_loaded
+                and self.breadcrumb.locations[0] == 'Scap Contents'
+                and not self.breadcrumb.read().starts_with('Edit ')
```

Comment:
that is redundant statement

Reply:
still you should understand that https://github.com/SatelliteQE/airgun/blob/master/airgun/navigation.py#L57

It will verify whether you have
```
                    and self.view.breadcrumb.locations[1] in (
                        entity_name,
                        'Edit {}'.format(entity_name))
                )
```
entity_name or 'Edit entity_name' in your breadcrumb automatically, so you do not need to specify that explicitly